src/main/java/net/pms/dlna/DLNAMediaInfo.java

```diff
@@ -1558,6 +1564,8 @@ public void postParse(int type, InputFile f) {
 				} else if (codecA.contains("pcm") || codecA.contains("wav") || codecA.contains("dts")) {
 					mimeType = HTTPResource.AUDIO_WAV_TYPEMIME;
 				} else if (codecA.contains("aac")) {
+					mimeType = HTTPResource.AUDIO_AAC_TYPEMIME;
```

Comment:
This is a bug, the AAC mime type only applies to raw AAC streams and not if AAC is in a container.